data/types/food/food.go

```diff
@@ -97,6 +113,19 @@ func (f *Food) Validate(validator structure.Validator) {
 	if f.Nutrition != nil {
 		f.Nutrition.Validate(validator.WithReference("nutrition"))
 	}
+	if f.Meal != nil && *f.Meal == MealRescueCarbs {
+		if f.Prescriptor != nil {
+			validator.String("prescriptor", f.Prescriptor).OneOf(Presciptors()...)
+		}
+
+		if *f.Prescriptor == HybridPrescriptor && f.PrescribedNutrition == nil {
+			// Prescribed Nutrition is mandatory
+			validator.WithReference("prescribedNutrition").ReportError(structureValidator.ErrorValueNotExists())
+		}
```

Comment:
Sending rescuecarbs meal with a nil prescriptor raises a 500 Error. Due to this test `*f.Prescriptor == HybridPrescriptor`.

```suggestion
		if f.Prescriptor != nil {
			validator.String("prescriptor", f.Prescriptor).OneOf(Presciptors()...)
			if *f.Prescriptor == HybridPrescriptor && f.PrescribedNutrition == nil {
				// Prescribed Nutrition is mandatory
				validator.WithReference("prescribedNutrition").ReportError(structureValidator.ErrorValueNotExists())
			}
		}
```

Reply:
applied in 024d276